custom_components/adaptive_lighting/switch.py

```diff
@@ -806,9 +816,9 @@ async def _adapt_light(
             service_data[ATTR_TRANSITION] = transition
 
         # The switch might be off and not have _settings set.
-        self._settings = self._sun_light_settings.get_settings(
+        self._settings.update(self._sun_light_settings.get_settings(
```

Comment:
Why is this now an `.update` instead of overwriting the original reference?

Reply:
Because we don't want what's in ._settings being overridden solely with values from the SunLightSettings() class. We merely want get_settings to update the values already inside _settings.

Reply:
Further research into an unrelated issue has me thinking that home assistant stores past values in attributes even if we don't send the key index with null, so this could possibly be left in as an =

Reply:
Resolved - changed to use a different dict to store the config